pinot-common/src/main/java/org/apache/pinot/common/response/BrokerResponse.java

```diff
@@ -137,6 +138,18 @@ String toJsonString()
    */
   int getExceptionsSize();
 
+  /**
+   * set the result table.
+   * @param resultTable result table to be set.
+   */
+  void setResultTable(ResultTable resultTable);
```

Comment:
Suggest not adding the setter to the interface

Reply:
setting is actually being used.